Updated ppa link with Canonical's feedback

juhlee-microsoft opened the pull request:
The permanent location for the PPA is:
https://launchpad.net/~canonical-server/+archive/ubuntu/dpdk-azure-18.11
with updated libraries:
- rdma-core v24
- dpdk 18.11.2
- openvswitch 2.11.1


Azure-Functional-INFINIBAND.INFINIBAND-INTEL-MPI-2VM   - PASSED
Azure-Functional-INFINIBAND.INFINIBAND-MVAPICH-MPI-2VM   - PASSED
Azure-Functional-INFINIBAND.INFINIBAND-OPEN-MPI-2VM   - PASSED

ader1990 commented:
Execution tested:

BEFORE:

```
[hyperv] 01/30/2019 11:44:19 : [INFO ] Checking if Hyper-V VM group 'LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295' exists on LIS-F2322...
[hyperv] 01/30/2019 11:44:19 : [INFO ] Stop-VM -Name LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm -Force -TurnOff
[hyperv] 01/30/2019 11:44:21 : [WARN ] Failed to remove snapshots for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm. Retrying...
[hyperv] 01/30/2019 11:44:21 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:23 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:25 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:27 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:29 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:31 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:33 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:35 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:37 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:39 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:41 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:43 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:45 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:47 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:49 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:51 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:53 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:56 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:44:58 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:00 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:02 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:04 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:06 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:08 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:10 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:12 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:14 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:16 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:18 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:20 : [INFO ] Waiting for VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm to enter 'Operating Normally' status
[hyperv] 01/30/2019 11:45:22 : [ERROR] EXCEPTION: VM LISAv2-TwoVM1Dep-hyper-v-LN30-636844149295-dependency-vm failed to enter Operating Normally status
[hyperv] 01/30/2019 11:45:22 : [ERROR] Source: Line 849 in script C:\jenkins_azure\workspace\peline-lisav2-hyperv_PR-303-VAPCAP6KMRVOM2ZGCXVJDNKZKQVOUEFNKAFE4VKKA4SSOB2NOVCA\Libraries\HyperV.psm1.
[hyperv] 01/30/2019 11:45:22 : [INFO ] Copying all files back to original working directory: C:\jenkins_azure\workspace\peline-lisav2-hyperv_PR-303-VAPCAP6KMRVOM2ZGCXVJDNKZKQVOUEFNKAFE4VKKA4SSOB2NOVCA.
[hyperv] Copy-Item : There is not enough space on the disk.
[hyperv] At C:\jenkins_azure\workspace\peline-lisav2-hyperv_PR-303-VAPCAP6KMRVOM2ZGCXVJDNKZKQVOUEFNKAFE4VKKA4SSOB2NOVCA\Librarie
[hyperv] s\WorkingSpaceManagement.psm1:50 char:2
[hyperv] +     Copy-Item -Path "$currentWorkingDirectory\*" -Destination $tmpDes ...
[hyperv] +     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[hyperv]     + CategoryInfo          : WriteError: (LISAv2-TwoVM1De...703FBB313.avhdx:FileInfo) [Copy-Item], IOException
[hyperv]     + FullyQualifiedErrorId : CopyDirectoryInfoItemIOError,Microsoft.PowerShell.Commands.CopyItemCommand
[hyperv]  
[hyperv] Copy-Item : There is not enough space on the disk.
[hyperv] At C:\jenkins_azure\workspace\peline-lisav2-hyperv_PR-303-VAPCAP6KMRVOM2ZGCXVJDNKZKQVOUEFNKAFE4VKKA4SSOB2NOVCA\Librarie
[hyperv] s\WorkingSpaceManagement.psm1:50 char:2
[hyperv] +     Copy-Item -Path "$currentWorkingDirectory\*" -Destination $tmpDes ...
[hyperv] +     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[hyperv]     + CategoryInfo          : WriteError: (LISAv2-TwoVM1De...C157CB106.avhdx:FileInfo) [Copy-Item], IOException
[hyperv]     + FullyQualifiedErrorId : CopyDirectoryInfoItemIOError,Microsoft.PowerShell.Commands.CopyItemCommand
[hyperv]  
[hyperv] 01/30/2019 11:49:38 : [INFO ] Setting workspace back to original location: C:\jenkins_azure\workspace\peline-lisav2-hyperv_PR-303-VAPCAP6KMRVOM2ZGCXVJDNKZKQVOUEFNKAFE4VKKA4SSOB2NOVCA
[hyperv] 01/30/2019 11:49:38 : [INFO ] Forcefully exiting with exit code 0 as ExitWithZero flag was set to True
[hyperv] 01/30/2019 11:49:38 : [INFO ] LISAv2 exit code: 0
```

AFTER:
```
[hyperv] 01/30/2019 16:43:31 : [INFO ] Checking if Hyper-V VM group 'LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676' exists on LIS-F2322...
[hyperv] 01/30/2019 16:43:31 : [INFO ] Stop-VM -Name LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-controller-vm -Force -TurnOff
[hyperv] 01/30/2019 16:43:33 : [INFO ] VHD C:\LISAv2\VH29\VHDs_Destination_Path\LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-controller-vm-diff-OSDisk_49A0AD4F-EBA9-490E-95C4-DE28215688B3.avhdx removed!
[hyperv] 01/30/2019 16:43:33 : [INFO ] VHD C:\LISAv2\VH29\VHDs_Destination_Path\LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-controller-vm-diff-OSDisk.vhdx removed!
[hyperv] 01/30/2019 16:43:33 : [INFO ] VHD C:\Users\lislab\AppData\Local\Temp\ResourceDisk-636844330838121384-sdb_3C7E1F1A-34EF-46FB-BE7A-C81E95D66536.avhdx removed!
[hyperv] 01/30/2019 16:43:33 : [INFO ] VHD C:\Users\lislab\AppData\Local\Temp\ResourceDisk-636844330838121384-sdb.vhdx removed!
[hyperv] 01/30/2019 16:43:33 : [INFO ] Hyper-V VM LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-controller-vm removed!
[hyperv] 01/30/2019 16:43:33 : [INFO ] Stop-VM -Name LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-dependency-vm -Force -TurnOff
[hyperv] 01/30/2019 16:43:39 : [INFO ] VHD C:\LISAv2\VH29\VHDs_Destination_Path\LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-dependency-vm-diff-OSDisk_AEC43311-8BF3-4F1F-92D9-E11A4A3ADE74.avhdx removed!
[hyperv] 01/30/2019 16:43:39 : [INFO ] VHD C:\LISAv2\VH29\VHDs_Destination_Path\LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-dependency-vm-diff-OSDisk.vhdx removed!
[hyperv] 01/30/2019 16:43:39 : [INFO ] VHD C:\Users\lislab\AppData\Local\Temp\ResourceDisk-636844331007662128-sdb_8675623A-69DC-4B77-A44F-360E4B3FCED0.avhdx removed!
[hyperv] 01/30/2019 16:43:39 : [INFO ] VHD C:\Users\lislab\AppData\Local\Temp\ResourceDisk-636844331007662128-sdb.vhdx removed!
[hyperv] 01/30/2019 16:43:40 : [INFO ] Hyper-V VM LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676-dependency-vm removed!
[hyperv] 01/30/2019 16:43:41 : [INFO ] Hyper-V VM group LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676 removed!
[hyperv] 01/30/2019 16:43:41 : [INFO ] Successfully delete HyperV group LISAv2-TwoVM1Dep-hyper-v-VH29-636844330676.
[hyperv] 01/30/2019 16:43:41 : [INFO ] Test VH29 finished
[hyperv] 01/30/2019 16:43:41 : [INFO ] 
[hyperv] [LISAv2 Test Results Summary]
[hyperv] Test Run On           : 01/30/2019 16:17:47
[hyperv] VHD Under Test        : \\redmond\wsscfs\OSTC\LIS\VHD\Cloudbase\Ubuntu\ubuntu_18.04.1\ubuntu_18.04.1.vhdx
[hyperv] Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
[hyperv] Total Time (dd:hh:mm) : 0:0:25
[hyperv] 
[hyperv]    ID TestCaseName                                                                TestResult TestDuration(in minutes) 
[hyperv] ---------------------------------------------------------------------------------------------------------------------
[hyperv]     1 NET-COPY-BOTH-WAYS                                                                PASS                22.83 
[hyperv] 
[hyperv] 
[hyperv] Logs can be found at C:\LISAv2\VH29\TestResults\2019-30-01-08-17-39-6760
```

juhlee-microsoft committed:
Updated ppa link with Canonical's feedback

The permanent location for the PPA is:
https://launchpad.net/~canonical-server/+archive/ubuntu/dpdk-azure-18.11
with updated libraries:
- rdma-core v24
- dpdk 18.11.2
- openvswitch 2.11.1

juhlee-microsoft committed:
Added add-apt-repo checking condition
